### PR TITLE
Update settings.gradle

### PR DESCRIPTION
**Pull request by shoatman**
As discussed over email:

- Comment out authenticator projects
- Remove MSAL CPP projects which no exist in MSAL CPP

### CODE REVIEW

File: settings.gradle [hunk 1]
```diff
@@ -13,7 +13,7 @@ rootProject.name = 'android_auth'
 include ':adal', ':msal', ':common', ':AADAuthenticator',
         ':brokerHost', ':adalTestApp', ':msalTestApp', ':package-inspector', ':pop-benchmarker',
         ':labapi', ':keyvault', ':testutils', 'msalautomationapp', 'uiautomationutilities', 'brokerautomationapp',
-        ':oneauth', ':oneauthtestapp', ':msalcpp', ':msalcpptestapp', ':msalcpptesthost'//, ':tsl', ':AzureSample',
+        ':oneauth', ':oneauthtestapp', ':msalcpptesthost'//, ':tsl', ':AzureSample',
```
Comment (shahzaibj) [inline settings.gradle:16]: I think we can bring back the AzureSample app

**Review by shahzaibj — APPROVED** [review]

File: settings.gradle (hunk 1 above)
Comment (shoatman) [reply settings.gradle:16]: It never works for me, so i think i need to figure out what's broken with it before I do that.

Comment (shahzaibj) [reply settings.gradle:16]: I have an open PR: https://github.com/Azure-Samples/ms-identity-android-java/pull/45

Comment (shoatman) [reply settings.gradle:16]: Approved.  Thanks!

**Pull request merged**